cibuildwheel/__main__.py

```diff
@@ -368,7 +418,10 @@ def detect_warnings(platform: str, build_options: BuildOptions) -> List[str]:
         option_value = getattr(build_options, option_name)
 
         if option_value and ('{python}' in option_value or '{pip}' in option_value):
-            warnings.append(option_name + ": '{python}' and '{pip}' are no longer needed, and will be removed in a future release. Simply use 'python' or 'pip' instead.")
+            warnings.append(
+                option_name
+                + ": '{python}' and '{pip}' are no longer needed, and will be removed in a future release. Simply use 'python' or 'pip' instead."
+            )
```

Comment:
I'm happy having this on one line. All you need when scrolling through the code is there: you're appending a warning. If part of the line is off screen, eh, who cares unless you're in the rare case of editing the warning message?

Reply:
this should be `msg = ...`, `warnings.append(msg)`, for consistency with raise statements. Then I think it would probably go on one line.